src/utils/formatLocalisedCompactNumber.ts

```diff
@@ -0,0 +1,12 @@
+import { getLanguageCodeFromLS } from 'contexts/Localization/helpers'
+
+const formatLocalisedCompactNumber = (number: number): string => {
+  const codeFromStorage = getLanguageCodeFromLS()
+  return new Intl.NumberFormat(codeFromStorage, {
```

Comment:
Helper for compact number formatting

![Screenshot 2021-07-26 at 16 20 04](https://user-images.githubusercontent.com/79279477/127231101-772c9cdd-df7a-482a-8502-7fb7bacde870.png)